fontforge/print.c

```diff
@@ -1155,23 +1156,11 @@ static void dump_pdfprologue(PI *pi) {
 	fprintf( pi->out, "  /Title (Character Displays from %s)\n", pi->mainsf->fullname );
     fprintf( pi->out, "  /Creator (FontForge)\n" );
     fprintf( pi->out, "  /Producer (FontForge)\n" );
-    time(&now);
-    tm = localtime(&now);
+    now = GetTime();
+    tm = gmtime(&now);
     fprintf( pi->out, "    /CreationDate (D:%04d%02d%02d%02d%02d%02d",
 	    tm->tm_year+1900, tm->tm_mon+1, tm->tm_mday, tm->tm_hour, tm->tm_min, tm->tm_sec );
-#ifdef _NO_TZSET
     fprintf( pi->out, "Z)\n" );
-#else
```

Comment:
Why is this removed?

Reply:
That part of the code was there only to calculate and output the timezone offset. With no timezone there is no need to calculate it. ;)

BTW, I think it is broken on MacOS (can't remember the details, I had a brief look before realizing it could just be deleted).